### ISSUE TITLE
Accessibility:List: The tooltip “Title1” provided for buttons, does not convey meaningful information about the UI

### ISSUE THREAD

**Issue by puru-hk** [SAP/fundamental-ngx#3477]
Description: The tooltip “Title1” provided for buttons, does not convey meaningful information about the UI

Expected: Tooltip/text alternative provided for UI should be meaningful and should define purpose of UI. 

Screenshot

![image](https://user-images.githubusercontent.com/32538291/94647606-73abda80-030e-11eb-8e7c-e5587b5fc056.png)

**Comment by sKudum** [comment]
Duplicate of #3485 , hence closing it.
